engine/api/workflow/process_test.go

```diff
@@ -49,7 +49,7 @@ func TestGetWorkflowRunStatus(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		status := getWorkflowRunStatus(tc.success, tc.building, tc.fail, tc.stop)
+		status := getWorkflowRunStatus(nil, tc.success, tc.building, tc.fail, tc.stop)
```

Comment:
what is this first argument ?

AS far as i know here is the func:
`func getWorkflowRunStatus(nodesRunSuccess, nodesRunBuilding, nodesRunFailed, nodesRunStopped int) string`